src/components/navigation-search/navigation-search.tsx

```diff
@@ -107,11 +109,20 @@ export class NavigationSearch extends Component<SearchProps, SearchState> {
           }}
         />
         {searchQuery && (
-          <button
-            className={styles.clearIcon}
-            onClick={this._onClear}
-            tabIndex={1}
-          />
+          <div>
+            <Tooltip
+              label="Search in Video"
+              type="bottom"
+              className={styles.tooltip}>
+              <Fragment>
+                <button
+                  className={styles.clearIcon}
+                  onClick={this._onClear}
+                  tabIndex={1}
+                />
+              </Fragment>
```

Comment:
What does a Fragment help here? I think it is redundant...

Reply:
Unfortunately not